extensions/common/json_util.cc

```diff
@@ -20,12 +20,13 @@
 namespace Wasm {
 namespace Common {
 
-::nlohmann::json JsonParse(absl::string_view str) {
-  return ::nlohmann::json::parse(str, nullptr, false);
+::nlohmann::json JsonParse(absl::string_view str, const bool allow_exceptions) {
+  return ::nlohmann::json::parse(str, nullptr, allow_exceptions);
```

Comment:
Exceptions are not supported in Wasm. I think we should always disallow that to avoid confusion.